### DISCUSSION TITLE
The Ethical Algorithm book by Aaron Roth

### DISCUSSION THREAD

**Discussion by lisaquera** [OpenMined/courses#194] [Resources]
Hello, 
One of the course speakers, Aaron Roth from the University of Pennsylvania, has an excellent book with a chapter on privacy in the context of FATE (Fair, Accountable, Transparent, and Ethical) algorithms: https://books.google.com/books/about/The_Ethical_Algorithm.html?id=QmmtDwAAQBAJ&source=kp_book_description

Hope you enjoy it as much as I did!

**Comment by yemikifouly** [comment]
Thank you so much for sharing Lisa! 😀
